packages/codemods/src/codeMods/tests/utilities/propsUtilities.test.ts

```diff
@@ -0,0 +1,91 @@
+import { renameProp, findJsxTag } from '../../utilities';
+import { Project, SyntaxKind, JsxAttribute } from 'ts-morph';
+import { EnumMap } from '../../types';
+
+const project = new Project();
+project.addSourceFilesAtPaths(`${process.cwd()}/**/tests/mock/**/*.tsx`);
+
+const personaPropsFile = 'mPersonaProps.tsx';
+const personaSpreadPropsFile = 'mPersonaSpreadProps.tsx';
+const spinnerPropsFile = 'mSpinnerProps.tsx';
+
+const deprecatedPropName = 'onRenderCoin';
+const newPropName = 'onRenderAvatarCoin';
+
+/* Developer will provide a mapping of Enum Values, if necessary. */
+const spinnerMap: EnumMap = {
+  normal: 'medium',
+  large: 'large',
+};
+
+describe('Props Utilities Test (Persona)', () => {
+  it('[SANITY] can ID the correct file and get the JSX elements', () => {
+    const file = project.getSourceFileOrThrow(personaPropsFile);
+    const tags = findJsxTag(file, 'Persona');
+    expect(tags.length).toEqual(2);
+  });
+
+  it('can rename props in a primitive', () => {
+    const file = project.getSourceFileOrThrow(personaPropsFile);
+    const tags = findJsxTag(file, 'Persona');
+    renameProp(tags, deprecatedPropName, newPropName, undefined, undefined);
+    tags.forEach(tag => {
+      expect(tag.getAttribute('onRenderCoin')).toBeFalsy();
+    });
+  });
+
+  it('can rename props in a spread attribute', () => {
+    const file = project.getSourceFileOrThrow(personaSpreadPropsFile);
+    const tags = findJsxTag(file, 'Persona');
+    renameProp(tags, 'primaryText', 'Text', undefined, undefined);
+    tags.forEach(val => {
+      val.getAttributes().forEach(att => {
+        if (att.getKind() === SyntaxKind.JsxSpreadAttribute) {
+          att
+            .getFirstChildByKind(SyntaxKind.Identifier)
+            ?.getType()
+            .getProperties()
+            .forEach(prop => {
+              expect(prop.getName()).not.toEqual('primaryText');
+            });
+        }
+      });
+    });
+  });
+});
+
+describe('Props Utilities Test (Spinner)', () => {
+  it('[SANITY] can ID the correct file and get the JSX elements', () => {
+    const file = project.getSourceFileOrThrow(spinnerPropsFile);
+    const tags = findJsxTag(file, 'Spinner');
+    expect(tags.length).toEqual(2);
+  });
+
+  it('can replace props with changed values (enums)', () => {
+    const file = project.getSourceFileOrThrow(spinnerPropsFile);
+    const tags = findJsxTag(file, 'Spinner');
+    let oldEnumValues: string[] = [];
+    tags.forEach(tag => {
+      oldEnumValues.push(
+        (tag.getAttribute('type') as JsxAttribute)
+          .getFirstChildByKind(SyntaxKind.JsxExpression)
+          .getFirstChildByKind(SyntaxKind.PropertyAccessExpression)
+          .getLastChildByKind(SyntaxKind.Identifier)
+          .getText(),
+      );
+    });
+    /* Icky reverse call here -- we want O(1) removal */
+    oldEnumValues.reverse();
+    renameProp(tags, 'type', 'size', 'SpinnerSize', spinnerMap);
```

Comment:
Since you setup the mock file, you can probably have some expectations about where things will be so that you can more easily go straight to the attribute/thing that you care about and check that!